guide/source/software_developer/structure.rst

```diff
@@ -144,5 +144,5 @@ This is to facilitate transfer of data elements not available in the standard.
    possible. Often times you will not be the only one with the need for a
    particular element that was overlooked in the standard. By participating in the
    working group and lobbying for the elements you need you can enhance the value
-   of HPXML for all parties. If everyone extends HPXML in non-standard ways, the
-   value proposition of the standard is eroded. 
+   of HPXML for all parties. [If everyone extends HPXML in non-standard ways, the
+   value proposition of the standard is eroded.] --> THIS APPEARS TO BE A FAIRLY IMPORTANT POINT. SHOULD IT BE EXPANDED UPON? PERHAPS HIGHLIGHTED? 
```

Comment:
It is in a large orange box with the "Warning" heading when rendered. I suppose it could be bolded or something.